feature-libs/product/configurators/rulebased/components/attribute/product-card/configurator-attribute-product-card.component.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  Component,
+  EventEmitter,
+  Input,
+  OnDestroy,
+  OnInit,
+  Output,
+} from '@angular/core';
+import { Configurator } from '../../../core/model/configurator.model';
+import { FormControl } from '@angular/forms';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'cx-configurator-attribute-product-card',
+  templateUrl: './configurator-attribute-product-card.component.html',
+})
+export class ConfiguratorAttributeProductCardComponent
+  implements OnDestroy, OnInit {
+  quantity = new FormControl(1);
+  private subs: Subscription;
+
+  @Input() disabledAction: boolean;
+  @Input() multiSelect = false;
+  @Input() product: Configurator.Value;
+  @Output() handleSelect = new EventEmitter();
+  @Output() handleDeselect = new EventEmitter();
+
+  ngOnInit() {
+    this.subs = this.quantity.valueChanges.subscribe((value) => {
+      if (value === 0) {
```

Comment:
is the value `0` here or just `false`? if the latter is true, then you should go with `if (!value) { ... }`

Reply:
changed